.circleci/config.yml

```diff
@@ -1,5 +1,18 @@
 version: 2.1
 
+orbs:
+  slack: circleci/slack@4.4.2
+
+commands:
+  notify-build-failed:
+      steps:
+        - slack/notify:
+            channel: mobile-bots
+            event: fail
```

Comment:
If the build passes, will the Slack channel get any notification at all? It might be nice to get a notification there, without the `@here` mention, just for consistency. I don’t think it would be too much noise if it only runs nightly.

Reply:
Thanks for the suggestion, will take a look how to archive this